scipy/optimize/minpack.py

```diff
@@ -133,6 +133,20 @@ def fsolve(func, x0, args=(), fprime=None, full_output=0,
     -----
     ``fsolve`` is a wrapper around MINPACK's hybrd and hybrj algorithms.
 
+    Examples
+    --------
+    We can find the roots of a function.
```

Comment:
This is a very generic statement, so it doesn't really add much to the example.  I suggest replacing it with something like

    Find a solution to the system of equations ``x0*cos(x1) = 4,  x1*x0 - x1 = 5``.

Reply:
Thank you for reviewing. I fixed it.